speaker/main.go

```diff
@@ -51,6 +52,8 @@ var announcing = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 	"ip",
 })
 
+var defaultConfigPath = "/etc/metallb/conf.yaml"
```

Comment:
nit: this doesn't have to be a var, could be a const. Also, I'd use the presence of the file instead of the boolean to understand if we need to enable the mechanism. Maybe that's where we want to override the path? But I think it's fair to have it hardcoded.

Also also, I'd do all the parsing in one shot and have the parse function return the regexp, so we have all the logic in one place.

Reply:
It would be better if we provide a flag to override the path, It's good for `helm install`：

when the value is empty in values.yaml: We don't create configMap, and we don't mount the file...

When the value is not empty and the file exists, we enable this mechanism， Is it ok?

Reply:
In addition , I think all the configurations here(https://github.com/metallb/metallb/blob/6a5ad674b791ca00ab6f797023a2a04d5b7903aa/speaker/main.go#L205), should be declared in one place so that it is easy to configure and update, how about this?

Reply:
I still don't see the value in setting passing the name as a parameter. It can be a convention.
If we don't want the file, we don't create the volume and the configmap, the load of the file will fail and we won't override.

Reply:
> I don't like this approach, having flags give a better understanding of what's being passed to the process and that's what people expect. Configmaps for knobs are fine if they are tweaks that are not normally expected to be used.

OK, Thanks for the reply.  I just had a whim, please ignore my thoughts

Reply:
>  If we don't want the file, we don't create the volume and the configmap, the load of the file will fail and we won't override.

 I currently read the env to determine if this mechanism is enabled and if the value is not empty and the file exists then it is enabled. The `-set .speaker.excludeInterfaces.configPath` flag is used when the `helm install`. The default configPath is '/etc/metallb/conf.yaml', if it is empty, it is not enabled.

Do you mean we don't need to expose env to enable this mechanism?

Reply:
> I still don't see the value in setting passing the name as a parameter.

The benefit of this is that we can decide whether to enable this mechanism when `helm install`

Reply:
we can still decide, we set a parameter in helm that drives the creation of the volume and the related configmap (and by default I'd create them). If those are not created, the go code will land in the notexists error and will not parse the map. Or I am missing something?

Reply:
Yes, That's what the PR does. The difference is that now this parameter is a string, indicating the config path to the file. Or should this be a boolean type? Then we hardcode the config file path in go code.